docs/linking/2-request-consent-web.puml

```diff
@@ -10,6 +10,11 @@ end box
 
 participant DFSP
 
+participant DFSPAuthorizeSimulator
+participant WebBrowser
+
+actor User
```

Comment:
Can `WebBrowser` and `User`  please be on the far left? That will line up better with our other sequences

Reply:
done

Reply:
Hmm I don't think the svg rerendered

Reply:
and now? I see the latest svg file